### DISCUSSION TITLE
Deploy Failed

### DISCUSSION THREAD

**Discussion by Dev-Lopez02** [serenity-is/Serenity#6491] [Help] [answered]
I made a deploy but when I go to the server, and enter the new modules, I get this error, I have searched everywhere and generated several times and still the same. 

![image](https://user-images.githubusercontent.com/108413888/183318521-5238224f-75fc-4a9a-a2ec-18163828ebee.png)

<html>
<body>
<span style="color: rgb(0, 0, 0); font-family: Verdana; font-size: 11.2px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;"><h1 style="font-family: Verdana; font-weight: normal; font-size: 18pt; color: red;">Error de servidor en la aplicación '/'.<hr width="100%" size="1" color="silver"></h1><h2 style="font-family: Verdana; font-weight: normal; font-size: 14pt; color: maroon;"><i>Invalid object name 'SegTrans.Modalidad'.</i></h2></span><span style="color: rgb(0, 0, 0); font-family: Verdana; font-size: 11.2px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(255, 255, 255); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial; display: inline !important; float: none;"></span><font face="Arial, Helvetica, Geneva, SunSans-Regular, sans-serif " style="color: rgb(0, 0, 0); font-size: 11.2px; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;"><b style="font-family: Verdana; font-weight: bold; color: black; margin-top: -5px;">Descripción:<span> </span></b>Excepción no controlada al ejecutar la solicitud Web actual. Revise el seguimiento de la pila para obtener más información acerca del error y dónde se originó en el código.<br><br><b style="font-family: Verdana; font-weight: bold; color: black; margin-top: -5px;">Detalles de la excepción:<span> </span></b>System.Data.SqlClient.SqlException: Invalid object name 'SegTrans.Modalidad'.<br><br><b style="font-family: Verdana; font-weight: bold; color: black; margin-top: -5px;">Error de código fuente:</b><br><br>

Se ha generado una excepción no controlada durante la ejecución de la solicitud Web actual. La información sobre el origen y la ubicación de la excepción pueden identificarse utilizando la excepción del seguimiento de la pila siguiente.
--

</font>
</body>
</html>

**Comment by edwardch** [comment, marked as answer]
To be certain, can you clean your solution and then run it in Release mode. And maybe check the case of the objects while you are at it. 
If that's all good then can you confirm all dlls and objects were uploaded correctly to the server?
